activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Move `db:migrate:status` code to `ActiveRecord::Tasks::DatabaseTasks.migrate_status`.
+
+    *Gannon McGibbon*
```

Comment:
Can you remove this changleog entry? I don't think it's necessary (rake task acts the same) and I think this may make users think that they can't use `db:migrate:status` anymore.

Reply:
👍 Removed!